Demo/Tests/ReactiveCocoaMoyaProviderTests.swift

```diff
@@ -60,10 +60,10 @@ class ReactiveCocoaMoyaProviderSpec: QuickSpec {
             }
 
             class TestProvider<Target: TargetType>: ReactiveCocoaMoyaProvider<Target> {
-                init(endpointClosure: @escaping EndpointClosure = MoyaProvider.DefaultEndpointMapping,
-                    requestClosure: @escaping RequestClosure = MoyaProvider.DefaultRequestMapping,
-                    stubClosure: @escaping StubClosure = MoyaProvider.NeverStub,
-                    manager: Manager = MoyaProvider<Target>.DefaultAlamofireManager(),
+                init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
+                    requestClosure: @escaping RequestClosure = MoyaProvider.defaultRequestMapping,
+                    stubClosure: @escaping StubClosure = MoyaProvider.neverStub,
+                    manager: Manager = MoyaProvider<Target>.defaultAlamofireManager(),
```

Comment:
Hmm, should we also consider making `defaultAlamofireManager` a variable? Looking by the init this is the only place we use brackets.